qemu/tests/cfg/usb.cfg

```diff
@@ -266,6 +266,24 @@
                 check_serial_option = no
                 check_removable_option = no
                 check_io_size_option = no
+            variants:
+                - @default:
+                    with_hotplug_unplug = no
+                - hotplug_unplug_1:
```

Comment:
Can we change the hotplug_unplug_1 to usb_one_time like line 295 to line 300 under usb_host
e.g.
variants:
         - usb_one_time:
               usb_repeat_times = 1
         - usb_multi_times:
               usb_repeat_times = 300

Reply:
Updated, thanks!